### ISSUE TITLE
Cable coils don't produce the correct cable cuff subtype for it's color

### ISSUE THREAD

**Issue by ShizCalev** [tgstation/tgstation#29821]
Repo:

1. Get a stack of cyan cabling (either /obj/item/stack/cable_coil/random or /obj/item/stack/cable_coil/cyan)
2. Make cable cuffs with said stack of cable
3. end up with red ones (/obj/item/weapon/restraints/handcuffs/cable) instead of the properly colored subtype, /obj/item/weapon/restraints/handcuffs/cable/cyan

This is true for all colors of cable.

Was planning on adding proper inhand sprites for each color variant, but there isn't much point of they aren't actually working anyway.

**Comment by PraiseRatvar** [comment]
i found more cable bugs

- When you take **one** cable out of a cable coil stack of 30 it gives you the wrong sprite (it gives the regular cable coil sprite instead of the cable piece sprite)

- it gives the wrong inhand (its always red) 

- it gives you a random color (if you take one cable out of a **green** cable coil stack it'll give you a random color e.g. white)

tldr; cable coils are stupid

**Comment by kingofkosmos** [comment]
>i found more cable bugs
>    When you take one cable out of a cable coil stack of 30 it gives you the wrong sprite (it gives the regular cable coil sprite instead of the cable piece sprite)
>    it gives the wrong inhand (its always red)
    it gives you a random color (if you take one cable out of a green cable coil stack it'll give you a random color e.g. white)
Couldn't reproduce any of these, works as intended. I took 1 cable out of a pink coil and got 1 pink cable piece.
